City-of-Helsinki/drupal-module-helfi-navigation
# UHF-8522: Check target entity's view access for each menu link

## How to install
- `composer require drupal/helfi_navigation:dev-UHF-8522`
- `drush cr`
- Start Etusivu instance (Etusivu doesn't need these changes)

## How to test

- Create a new node with menu link and unpublish the node
- Run `drush cron` and make sure the menu link is not visible in Mega menu: 
![image](https://github.com/City-of-Helsinki/drupal-module-helfi-navigation/assets/771113/fb513232-9009-4069-8bf8-a48c9b8ab7a9)
- Publish the node, run `drush cron` again, `drush cr` and make sure the menu link is now visible in Mega menu
